axelor-supplychain/src/main/resources/views/Cart.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<object-views xmlns="http://axelor.com/xml/ns/object-views"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://axelor.com/xml/ns/object-views http://axelor.com/xml/ns/object-views/object-views_7.1.xsd">
+
+  <grid id="supplychain-cart-grid" name="cart-grid" title="Carts"
+    model="com.axelor.apps.sale.db.Cart" extension="true">
+    <extend target="//field[@name='partner']">
+      <insert position="after">
+        <field name="stockLocation"/>
+      </insert>
+    </extend>
+  </grid>
+
+  <form id="supplychain-cart-form" name="cart-form" title="Cart"
+    model="com.axelor.apps.sale.db.Cart" extension="true">
+    <extend target="//field[@name='partner']">
+      <insert position="after">
+        <field name="stockLocation" form-view="stock-location-form"
```

Comment:
is there a way to reload the available status after updating the stock location ? If this is possible please do it